docs/minikube.md

```diff
@@ -0,0 +1,383 @@
+# Minikube installation
+
+
+For the purporse of testing Insprd locally we recommmend using [minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+Following this tutorial you will see the necessary steps to have one of the Inspr's examples running locally, more specifically the `ping pong` demonstration, but it will work for any other.
+
+The requirements for this turorial are:
+- Docker Hub account
+- Docker installed
+- kubectl installed
+- Helm installed
+- insprctl installed
+- Skaffold installed
+
+
+# Step by Step
+
+## Setting up minikube
+Initialize the cluster by running `minikube start` in your terminal. 
+
+Run `kubectl config get-contexts` to confirm that the currently selected cluster is minikube.
+
+After this is set, run the commands below for setting up the permissions of the cluster created and to create the namespace `inspr-apps`.
+
+```
+$ kubectl create clusterrolebinding --clusterrole admin --serviceaccount default:default defaultserviceaccount
+$ kubectl create namespace inspr-apps
+```
+
+## Installation of the insprd and uidp
+
+The process of installation involves adding a few Helm repositories and running `helm install`, the detailed process can be found [here](helm_installation.md).
+
+In short the process consists of the following commands:
+
+### **Installing NGINX**
+```
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+$ helm install ingress-nginx ingress-nginx/ingress-nginx
+```
+
+For confirmation that the service is up and running run the command below, it should be a service with the name `ingress-nginx-controller`.
+```
+$ kubectl get service -A
+```
+
+## **Installing Insprd**
+```bash
+$ helm repo add inspr https://inspr-charts.storage.googleapis.com/
+$ helm repo update
+$ helm install insprd inspr/insprd \
+    --set insprdRepository=gcr.io/red-inspr/insprd:v0.1.0 \
+    --set authRepository=gcr.io/red-inspr/authsvc:v0.1.0 \
+    --set secretRepository=gcr.io/red-inspr/secretgen:v0.1.0 \
+    --set lbsidecarImage=gcr.io/red-inspr/inspr/sidecar/lbsidecar:v0.1.0
+```
+
+## **Installing UIDP Provider**
+
+With the `insprd` and its `auth-svc` installed we can now install the inspr's UIDP Provider. This can be done by running `skaffold` in the [github repository](https://github.com/inspr/inspr) that contains the necessary information (you should clone the repository to your local machine).
```

Comment:
- Inspr* (with capital letter)  
- It's not "UIDP Provider", it's "UID Provider"